MentoringHomeWork/HW9/AirportSimulation/Countries.cs

```diff
@@ -0,0 +1,24 @@
+namespace AirportSimulation;
+
+public class Countries
```

Comment:
This class can be enum initially. 
In this case, you will be able to use just YourEnumClass in all places - not **Countries**._CountryPassengers_ -> just **CountryPassengers** - also about naming for Enum, usually it is in singular meaning, for example, Country (not **Countries**), PassportType (not **PassportTypes**)

I suggest changing Countries.cs -> PassengerCountry.cs (is it destination countries or where is the passenger from?)
```suggestion
namespace AirportSimulation;

public enum PassengerCountry
{
    Belarus,
    Russia,
    USA,
    Germany,
    France,
    Italy,
    Japan,
    China,
    Australia,
    UK,
    Canada,
    India,
    Brazil,
    SouthKorea,
    Spain,
    Mexico
}
```

Reply:
resolved